Makefile

```diff
@@ -87,8 +87,8 @@ rpm-release: srpm-release
 	mock --old-chroot --resultdir BUILD/RPM -D "commit $(RELEASE_COMMIT)" --rebuild BUILD/SRPM/avocado-plugins-vt-$(VERSION)-*.src.rpm
 
 requirements:
-	- pip install "pip>=6.0.1"
-	- pip install -r requirements.txt
+	- $(PYTHON) -m pip install "pip>=6.0.1"
```

Comment:
Good catch.  And  this is *not* yet on Avocado.  I'll send a fix there.

Reply:
Here it is:

https://github.com/avocado-framework/avocado/pull/2734

Basically your own changes and commit message.